### PR TITLE
add new deploy sl section

### PR DESCRIPTION
**Pull request by mirnawong1**
adding new 'deploy SL' section per PM feedback. Also added additional subfolder to categorize docs: configure, deploy, consume. 

created a new consume metrics page to orient users and provide them a clearer way to consume metrics via integrations/api/sdk.

### CODE REVIEW

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Another great PR! No suggestions or recommendations from me. I've approved this for you.

Kind Regards
Natalie

File: website/docs/docs/use-dbt-semantic-layer/deploy-sl.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Deploy your Semantic Layer"
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/deploy-sl.md]:
```suggestion
title: "Deploy your metrics"
```

File: website/docs/docs/use-dbt-semantic-layer/deploy-sl.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Deploy your metrics"
+id: deploy-sl
+description: "Deploy the dbt Semantic Layer in dbt Cloud by running a job to materialize your metrics."
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/deploy-sl.md:4]:
```suggestion
description: "Deploy the dbt Semantic Layer in dbt Cloud by running a job to materialize your metrics."
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-sl.md [hunk 3]
```diff
@@ -33,6 +33,12 @@ plan="dbt Cloud Team or Enterprise"
     link="/guides/sl-snowflake-qs"
     icon="dbt-bit"/>
 
+<Card
+    title="Deploy dbt Semantic Layer"
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-sl.md]:
```suggestion
    title="Deploy metrics"
```

File: website/docs/docs/use-dbt-semantic-layer/deploy-sl.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Deploy your metrics"
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/deploy-sl.md:2]: is deploy the right word here or is 'materialize' clearer?

File: website/docs/docs/use-dbt-semantic-layer/dbt-sl.md [hunk 5]
```diff
@@ -25,54 +26,28 @@ product="dbt Semantic Layer"
...
+- [Architecture](/docs/use-dbt-semantic-layer/sl-architecture) &mdash; Explore the powerful components that make up the dbt Semantic Layer.
 
-<Card
-    title="Available integrations"
-    body="Review a wide range of partners you can integrate and query with the dbt Semantic Layer."
-    link="/docs/cloud-integrations/avail-sl-integrations"
-    icon="dbt-bit"/>
+## Deploy metrics
+This section provides information on how to deploy the dbt Semantic Layer and materialize your metrics:
+- [Deploy your Semantic Layer](/docs/use-dbt-semantic-layer/deploy-sl) &mdash; Run a dbt Cloud job to deploy the dbt Semantic Layer and materialize your metrics.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-sl.md:44]:
```suggestion
- [Deploy your Semantic Layer](/docs/use-dbt-semantic-layer/deploy-sl) &mdash; Run a dbt Cloud job with `dbt parse` to deploy the dbt Semantic Layer and materialize your metrics.
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-sl.md [hunk 6]
```diff
@@ -25,54 +26,28 @@ product="dbt Semantic Layer"
...
+- [Architecture](/docs/use-dbt-semantic-layer/sl-architecture) &mdash; Explore the powerful components that make up the dbt Semantic Layer.
 
-<Card
-    title="Available integrations"
-    body="Review a wide range of partners you can integrate and query with the dbt Semantic Layer."
-    link="/docs/cloud-integrations/avail-sl-integrations"
-    icon="dbt-bit"/>
+## Deploy metrics
+This section provides information on how to deploy the dbt Semantic Layer and materialize your metrics:
+- [Deploy your Semantic Layer](/docs/use-dbt-semantic-layer/deploy-sl) &mdash; Run a dbt Cloud job with `dbt parse` to deploy the dbt Semantic Layer and materialize your metrics.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-sl.md]:
```suggestion
- [Deploy your Semantic Layer](/docs/use-dbt-semantic-layer/deploy-sl) &mdash; Run a dbt Cloud job to deploy the dbt Semantic Layer and materialize your metrics.
```

File: website/docs/docs/use-dbt-semantic-layer/dbt-sl.md [hunk 7]
```diff
@@ -15,7 +15,8 @@ Moving metric definitions out of the BI layer and into the modeling layer allows
 
 Refer to the [dbt Semantic Layer FAQs](/docs/use-dbt-semantic-layer/sl-faqs) or [Why we need a universal semantic layer](https://www.getdbt.com/blog/universal-semantic-layer/)  blog post to learn more.
 
-## Explore the dbt Semantic Layer
+## Get started
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/dbt-sl.md]:
```suggestion
## Get started with the dbt Semantic Layer
```

**Pull request merged**